### PR TITLE
11 adding pictures to bfs and dfs example

### PR DESCRIPTION
**Pull request by MaxGreenspan**
BFS's comparison section is identical to that of DFS's comparison section, except that doesn't have the necessary pictures that accompany it. I added these pictures and fixed a few minor spelling errors.

### CODE REVIEW

File: experiment/analysis/comparison-of-bfs-with-dfs.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
    - Similarly, E is also removed.
-   - Now, vertex B is at the top of the stack, its another node(vertex) F is explored and stored in the stack.
+   - Now vertex B is at the top of the stack, its other node(vertex) F is explored and stored in the stack.
    - Vertex F has two successors C and D, between which C is traversed first and stored in the stack.
-   - Vertex C has only one predecessor which is already visited, so it is removed from the stack.
-   - Now vertex D connected to F is visited and stored in the stack.
-   - As vertex D does not have any unvisited nodes, therefore D is removed.
-   - Similarly, F, B and A are also popped.
+   - Vertex C has only one predecessor which has already been visited, so it is removed from the stack.
+   - Now vertex D which is connected to F is visited and stored in the stack.
```
Comment (raj-vlabs) [inline experiment/analysis/comparison-of-bfs-with-dfs.md:37]: Should there be a comma after D:
Now vertex D, which is

File: experiment/analysis/comparison-of-bfs-with-dfs.md [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
 
    - Considering A as the starting vertex which is explored and stored in the stack.
    - B, the successor vertex of A, is stored in the stack.
-   - Vertex B have two successors E and F, among them alphabetically E is explored first and stored in the stack.
+   - Vertex B has two successors E and F, among them alphabetically E is explored first and stored in the stack.
    - The successor of vertex E, i.e., G is stored in the stack.
    - Vertex G has two connected vertices, and both are already visited, so G is popped out from the stack.
    - Similarly, E is also removed.
-   - Now, vertex B is at the top of the stack, its another node(vertex) F is explored and stored in the stack.
+   - Now vertex B is at the top of the stack, its other node(vertex) F is explored and stored in the stack.
```
Comment (raj-vlabs) [inline experiment/analysis/comparison-of-bfs-with-dfs.md:34]: Should we say, 'its other successor' in place of 'its other node' so we stay consistent with line 30 on the terminology?

File: experiment/analysis/comparison-of-bfs-with-dfs.md [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
+  -  At last C is also removed and the queue is now empty which means we are done.
   -  The generated Output is – A, B, D, G, E, F, C.
 
-### DFS traversal on Example graph
+  <img src="images/bfs.png"/>
 
-Similar to BFS lets take the same graph for performing DFS operations, and the involved steps are:
+### DFS traversal on example graph
+
+Similar to BFS lets take the same graph for performing DFS operations. The steps involved in the process are:
```
Comment (raj-vlabs) [inline experiment/analysis/comparison-of-bfs-with-dfs.md:26]: Should it be:
Similar to BFS, let's take the same graph for performing DFS operations

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
Please look at the few suggestions in the review comments and resolve them by either responding to them or by applying them.

File: experiment/analysis/comparison-of-bfs-with-dfs.md (hunk 3 above)
Comment (MaxGreenspan) [reply experiment/analysis/comparison-of-bfs-with-dfs.md:26]: Yeah, lets is supposed to be a contraction, and the comma adds clarity.

File: experiment/analysis/comparison-of-bfs-with-dfs.md (hunk 2 above)
Comment (MaxGreenspan) [reply experiment/analysis/comparison-of-bfs-with-dfs.md:34]: I agree that that is better than 'its other node'.

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**